xapian-bindings/python3/docs/examples.rst

```diff
@@ -8,7 +8,6 @@ simplesearch.py
 
 .. literalinclude :: examples/simplesearch.py
    :language: python
-   :emphasize-lines: 12,15-18
```

Comment:
Once fixups are collapsed, this change (and the similar ones below) is being made as part of a commit "Add letor bindings of python3 to xapian-bindings", but it has no connection to that change at all.  It is a change we want to make, but it really ought to be done separately (indeed I have a note to address this - I've held off from just doing this as it would probably have conflicted with this PR).

Reply:
Perhaps I should give some practical examples on why lumping something like this into an unrelated commit is unhelpful.

Currently the result of these `:emphasize-lines: 12,15-18` is an odd visual glitch in the HTML output, where those 5 lines get emphasised for no good reason.  That's something which it would be beneficial to fix in 1.4.x too - it's about a safe a change as they come, and improves the documentation a little.

But adding bindings for xapian-letor is not 1.4.x material at all.  Include this change here makes it harder to backport (because I can't just `git cherry-pick` a single commit and have it apply cleanly - I would need to pull out part of a commit, and even part of the changes made to one file by a commit).

It also makes it less obvious there's even something to backport.  I backport based on checking each commit on master and marking it as not to backport or backported - if a commit says "Add letor bindings of python3 to xapian-bindings" then I would likely quickly mark that as not to backport, and a change which ought to be backported gets missed.

Similar to backporting, a packager of xapian-bindings might want to pull out a particular fix and apply it to their package as an extra patch - that's much simpler to do if it's a single clean commit, and simpler to keep track of when the patch is safe to drop.

If on the other hand this extra change turned out to be wrong (which in this case seems very unlikely, but in general can happen) then we'd want to revert the change.  If it's done as a separate commit, then `git revert` will undo it, usually without needing manual intervention.  But if it's lumped in to another commit then again someone has to pick out parts of a commit, but this time to undo them rather than apply them to a different barnch - and that introduces more scope for errors.

Reply:
I have added a fixup commit which adds the emphasis lines and a new commit which removes it.
I hope that is good enough.

Reply:
Sounds OK.